docs/source/10-min.ipynb

```diff
@@ -92,15 +93,16 @@
    "source": [
     "You can also load DataFrames from other sources, such as:\n",
     "\n",
-    "1. CSV files: `daft.read_csv(\"s3://bucket/*.csv\")`\n",
-    "2. Parquet files: `daft.read_parquet(\"/path/*.parquet\")`\n",
-    "3. JSON line-delimited files: `daft.read_json(\"/path/*.parquet\")`\n",
-    "4. Files on disk: `daft.from_glob_path(\"/path/*.jpeg\")`\n",
+    "1. CSV files: {func}`daft.read_csv(\"s3://bucket/*.csv\") <daft.read_csv>`\n",
```

Comment:
What does this look like in the notebook?

Reply:
A bit less readable in-notebook, but we ultimately decided that the cross-referencing was worth it when writing up a bunch of examples for Ray's ML ecosystem. Ideally there would be a Jupyter plugin that would render the markdown properly in the notebook itself, but we never figured that out.

![Screenshot 2023-06-06 at 6 53 54 PM](https://github.com/Eventual-Inc/Daft/assets/7699919/326fa49a-3535-4f6c-9f9d-6db3136c2722)

Reply:
I'll let @jaychia plug in on this since he wrote it!

Reply:
@jaychia Any opposition to this? I'd like to merge this soon if possible!

Reply:
Talked to @jaychia offline and he says that he thinks this is a net-positive change! Merging now.

Reply:
Ship!! 🚢